website/docs/product/telemetry.md

```diff
@@ -8,58 +12,102 @@ Appsmith is a lightweight proxy and **does not capture** any data returned by yo
+        "$insert_id": "d684e7ed-1211-4518-a19c-8f22925526e0",
+        "$lib_version": "3.3.1",
+        "$mp_api_endpoint": "api.mixpanel.com",
+        "$mp_api_timestamp_ms": 1704784101374,
+        "$region": "Karnataka",
+        "$source": "segment",
+        "$user_id": "659cf0bd504813315c336bf7",
+        "disable-telemetry": false,
+        "email": "",
+        "emailDomainHash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+        "goal": "just exploring",
+        "id": "659cf0bd504813315c336bf7",
+        "instanceId": "659cf0bd504813315c336bf7",
+        "ipAddress": "117.211.16.3", // IP address is of the server used to host the instance and not the client IP
+        "mp_country_code": "IN",
+        "mp_lib": "Segment Actions: analytics-java",
+        "mp_processing_time_ms": 1704784101771,
+        "name": "",
+        "originService": "appsmith-server",
+        "proficiency": "",
+        "role": "frontend engineer",
+        "segment_source_name": "ce",
+        "subscribe-marketing": false, // If user subscribes to marketing updates, we get the name and email address as well along with above information
+        "version": "v1.9.56"
     }
-  },
-  "event": "execute_ACTION_TRIGGERED",
-  "integrations": {},
-  "messageId": "0f6b07ee-0717-413-808c-c25b09c0468",
-  "originalTimestamp": "2021-08-24T07:23:35.610Z",
-  "properties": {
-    "appId": "612465f87b2230debedfc6",
-    "appMode": "edit",
-    "appName": "APP1",
-    "datasource": {
-      "name": "Test App"
-    },
-    "instanceId": "612460418944011a10fa5b",
-    "isExampleApp": false,
-    "isSuccessfulExecution": true,
-    "name": "Test",
-    "orgId": "612464f7f230debedfc4",
-    "originService": "appsmith-server",
-    "pageId": "612465802230debedfc8",
-    "pageName": "Page1",
-    "pluginName": "PostgreSQL",
-    "statusCode": "",
-    "timeElapsed": 8,
-    "type": "DB",
-    "username": "70280e5d07e61e5e915e5d26ac8704bbd68d3f75ebad67ba439f4c354d7"
-  },
-  "receivedAt": "2021-08-24T07:23:39.996Z",
-  "sentAt": "2021-08-24T07:23:39.885Z",
-  "timestamp": "2021-08-24T07:23:35.721Z",
-  "type": "track",
-  "userId": "70280e5dd9e61e5e91526ac8704bbd68d3f75ebad67ba439f4c354d7",
 }
+
 ```
 
-### Navigation and clicks
+### Usage pulse for billing
 
-The client captures anonymous behavioral data around navigation and clicks.
+The Appsmith client triggers a usage pulse whenever a user performs an action on their Appsmith instance. The usage pulse primarily contains information such as the timestamp, a hashed `userId` to map the user, and the app mode in which the action was performed. The data is collected on the server-side and sent to the Appsmith cloud services in batches every hour irrespective of whether telemetry is turned on or off. These pulses are then processed to present aggregate information to customers on customer portal. This data is collected only for paying customers, regardless of whether telemetry is on or off.
```

Comment:
"regardless of whether telemetry is on or off" is repeated twice, can we remove one of them?